src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -3,37 +3,19 @@
   "name": {
     "text": "Worker Service",
     "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
-    "id": "1011"
+    "id": "1025"
   },
   "description": {
     "text": "An empty project template for creating a worker service.",
     "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
-    "id": "1012"
+    "id": "1026"
   },
   "order": 100,
   "icon": "vs-2017.3/Empty.png",
   "learnMoreLink": "https://go.microsoft.com/fwlink/?linkid=2072778",
   "uiFilters": [ "oneaspnet" ],
-  "supportsDocker": true,
+  "supportsDocker": false,
   "legacyTemplateIdentity": "Microsoft.NetCore.CSharp.EmptyWorker",
```

Comment:
By the way, you can kill the legacyTemplateIdentity, it was pretty much there to provide consistent telemetry when we swapped moved to the template engine based templates.